.travis.yml

```diff
@@ -58,6 +58,13 @@ jobs:
       script: git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && sbt whitesourceCheckPolicies whitesourceUpdate
       name: "Check licenses via WhiteSource"
 
+    - stage: publish
+      env: CMD="+publish"
+      name: "Publish artifacts to Bintray"
+    - script: openssl aes-256-cbc -K $encrypted_9f19d2edcfdd_key -iv $encrypted_9f19d2edcfdd_iv -in .travis/travis_apjdbc_rsa.enc -out .travis/id_rsa -d && eval "$(ssh-agent -s)" && chmod 600 .travis/id_rsa && ssh-add .travis/id_rsa && sbt -jvm-opts .jvmopts-travis docs/publishRsync
```

Comment:
Just a minor nitpick. WDYT about moving this long druidic spell to a bash script?

Reply:
Alos, just out curiosity, where do `$encrypted_9f19d2edcfdd_key` and `$encrypted_9f19d2edcfdd_iv` come from? I understand Travis injects those into the VM but are those values set on the Travis UI?

Reply:
If it works, I won't touch it again...

https://docs.travis-ci.com/user/encrypting-files/

Reply:
> If it works, I won't touch it again...

I think I have some room on my left arm to get that tattoo'd.